source/PyMaterialX/PyMaterialXGenShader/PyShaderGenerator.cpp

```diff
@@ -290,18 +290,6 @@ class PyShaderGenerator : public mx::ShaderGenerator
         );
     }
 
-    bool remapEnumeration(const mx::ValueElement& input, const std::string& value, std::pair<const mx::TypeDesc*, mx::ValuePtr>& result) const override
```

Comment:
Not used anywhere and should probably have never been exposed in the Python API.